src/pages/multisig/utils/placeholder.ts

```diff
@@ -1,5 +1,5 @@
 export const SAMPLE_ENCODED_TX =
-  "CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcXJqZHE3bWZkenl5Zmw2Y3RuOHZ0a3d0NDNndnl1dGxhEix0ZXJyYTE5NWxxanFyamRxN21mZHp5eWZsNmN0bjh2dGt3dDQzZ3Z5dXRsYRoQCgV1bHVuYRIHMTAwMDAwMBIVEhMKDQoFdWx1bmESBDE4NzIQ0IoK"
+  "CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcx(JqZHE3bWZkenl5Zmw2Y3RuOHZ0a3d0NDNndnl1dGxhEix0ZXJyYTE5NWxxanFyamRxN21mZHp5eWZsNmN0bjh2dGt3dDQzZ3Z5dXRsYRoQCgV1bHVuYRIHMTAwMDAwMBIVEhMKDQoFdWx1bmESBDE4NzIQ0IoK"
```

Comment:
Did you mean to add the `(` in the first line here?

CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcx  (  JqZHE3bWZke